integrations/serverless-cost-management-integration.adoc

```diff
@@ -6,19 +6,19 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/getting_started_with_cost_management/assembly-introduction-cost-management#about-cost-management_getting-started[Cost management] is an {ocp-product-title} service that enables you to better understand and track costs for clouds and containers. It is based on the open source link:https://project-koku.github.io/[Koku] project.
+link:https://docs.redhat.com/en/documentation/cost_management_service/1-latest/html/getting_started_with_cost_management/index[Cost management] is an {ocp-product-title} service that enables you to better understand and track costs for clouds and containers. It is based on the open source link:https://project-koku.github.io/[Koku] project.
```

Comment:
Global comment:
@ochromy and (my onboarding buddy for peer reviews) @dfitzmau, I've never seen such versioning before: `1-latest`. I've only seen `latest` used in docs URLs. Also, it takes a while to mentally process `1-latest` in the doc title, because usually product versions are just numbers. BTW, when version 1.y or 2 is released, will version `1-latest` be renamed to version `1` and version 2 be named `2-latest`?

Reply:
Hi @max-cx . Good to query this for sure. Serverless might have a different approach on how they document versioning. Let's wait for a reply to confirm this.

Reply:
First time I've seen this system as well. 
The `1-latest` is currently the only available version of the Insights cost management docs. 
It is the same with, for example, Edge management or Subscription Central.